### ISSUE TITLE
QEditor fontSize default

### ISSUE THREAD

**Issue by Sweetyy** [quasarframework/quasar#6505]
Hi!

Referring to [this post](https://forum.quasar-framework.org/topic/5364/qeditor-fontsize-default) on  the forum, the QEditor with Quasar 1.9.4 has some font size match errors:

- By default it’s **A bit small** (in the selector) that seems to be set to the text we write. But after clicking on **A bit small**, font size decreases even more.
- In addition if you select **Normal** size, it will give you a bigger size than default one.
- Same problem when you **Remove the format** of the text.

To sum up, I think there are issues with the font by default in the QEditor and the tags `<font size="*"></font>` added in the QEditor after selecting a specific size.

Thanks a lot for the awesome work!

**Comment by pdanpdan** [comment]
We will not implement this in core. Please use an external html editor for more complex things.

**Comment by mijaelsaban** [comment]
oh no, is there some update on this issue?
the heading 1 looks enormous

thank you

**Comment by mijaelsaban** [comment]
I don't know if there is a proper solution
fixed like this for now

```
`div.q-item__section.column.q-item__section--main.justify-center > div > h1 {
    font-size: 3rem;
}

div.q-item__section.column.q-item__section--main.justify-center > div > h2 {
    font-size: 2.5rem;
}

div.q-item__section.column.q-item__section--main.justify-center > div > h3 {
    font-size: 1.5rem;
}

div.q-editor__content.overflow-auto > h1 {
    font-size: 3rem;
}

div.q-editor__content.overflow-auto > h2 {
    font-size: 2.5rem;
}

div.q-editor__content.overflow-auto > h3 {
    font-size: 1.5rem;
}`
```
